packages/optimizer/lib/DomTransformer.js

```diff
@@ -145,13 +144,13 @@ class DomTransformer {
    * @param {Array.<Transformer>} config.transformations - a list of transformers to be applied.
    */
   setConfig(config) {
```

Comment:
I might opt for `this.config = this.setConfig(config);` in the constructor so that it's easier to identify that DomTransformer has property `config` available without digging into a function.

Reply:
This duplicates initializing the field in the constructor and in the setter. Imo setX implies that there's a field (my Java roots might be showing trough here...)